lib/chef/org/group_operations.rb

```diff
@@ -0,0 +1,60 @@
+require_relative "../org"
+
+class Chef
+  class Org
+    module GroupOperations
```

Comment:
I wonder if we really need a separate module for this.  It was structured this way in the plugin because we lived outside the Chef codebase, but we could probably add these features directly to the Chef::Org class or create a new Chef::Group class that would abstract away some of these features.